From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: insilications <boboniboni@gmail.com>
Date: Sun, 2 Aug 2020 03:12:44 -0300
Subject: [PATCH] Add Clear Linux cert_path

Change '/etc/pki/ca-trust' to the correct CL path.
---
 mock/py/mockbuild/package_manager.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mock/py/mockbuild/package_manager.py b/mock/py/mockbuild/package_manager.py
index 1ba26da..cb6c391 100644
--- a/mock/py/mockbuild/package_manager.py
+++ b/mock/py/mockbuild/package_manager.py
@@ -301,7 +301,7 @@ Error:      Neither dnf-utils nor yum-utils are installed. Dnf-utils or yum-util
 
     @traceLog()
     def copy_certs(self):
-        cert_path = "/etc/pki/ca-trust"
+        cert_path = "/var/cache/ca-certs/anchors"
         pki_dir = self.buildroot.make_chroot_path(cert_path)
         try:
             copy_tree(cert_path, pki_dir)
